pkg/common/error.go

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
No need to create a new file just for a single error variable. You can move it to `utils.go`.